.github/workflows/release.yml

```
@@ -36,44 +38,47 @@ jobs:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_URL: ${{ env.SENTRY_URL }}
         run: |
-          # Extract version from tag
-          VERSION=${GITHUB_REF#refs/tags/v}
+          set -euo pipefail  # Bash safety flags
+          
+          # Extract version and create consistent release name (strip 'v' prefix)
+          VERSION="${{ github.ref_name }}"
+          VERSION=${VERSION#v}  # Remove 'v' prefix if present
+          RELEASE="${{ env.SENTRY_PROJECT }}@$VERSION"
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)